app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
We need to support Pairwise classifications like the Sunspotters project at some point. This would get us out ahead like that. 

And yeah we could just add a task in the background worker that updates a counter and checks the retirement rules. I think moving towards using Kafka and Storm or Spark for most of our background tasks will make that migration easier in the future when have to. 

For this I'd probably make a quick Storm topology and store the counts in Redis or DynamoDB get something up and running quickly. The topology could also check retirement conditions.